docs/src/tut/wfs/index.md

```diff
@@ -0,0 +1,1343 @@
+		const acc = acc.contract(backend);
+		await backend.Seller(ctc, sellerInteract);
+	}
+	
+	// Buyer
+	else {
+		const buyerInteract = { ...commonInteract };
+	;
+
+})();
+```
+
+Below is a line-by-line description of this JavaScript code:
+
+* Line 1: Import the Reach JS Standard Library loader.
+* Line 2: Import the JS backend compiled from index.rsh.
+* Line 4: Hard-code the role. You will change this later.
+* Line 5: Display the role.
+* Line 7: Load the Reach JS Stdlib for the consensus network specified by `REACH_CONNECTOR_MODE` env var.
+* Line 8: Display the consensus network type.
+* Line 10: Enable enclosed code to await the fulfillment of promises.
+* Line 12: Define empty (for now) object.
+* Line 15: Code for when you run this app as the seller.
+* Line 16: Define empty (for now) object.
+* Line 18: Create an account for the seller. `parseCurrency` transforms units from standard to atomic.
+* Line 19: Get a reference to the contract.
+* Line 20: Initiate interaction with contract for seller.
+* Line 24: Code for when you run this app as the buyer.
+* Line 25: Define empty (for now) object.
+
+## {#wfs-5} Review backend starter
+
+Copy the following code into the file named `index.rsh`.
+
+```reach
+'reach 0.1';
+
+const commonInteract = {};
+const sellerInteract = { ...commonInteract };
+const buyerInteract = { ...commonInteract };
+
+export const main - Reach.App()) => {
+	const S = Participant('Seller', sellerInteract);
+	const B = Participant('Buyer', buyerInteract);
+	init();
+	
+	exit()
+]);
+```
+
+Below is a line-by-line description of this `reach` code:
+
+* Line 1: Instruction to the compiler.
+* Lines 3-5: Define empty (for now) objects.
+* Line 7: Reach standard application initialization.
+* Line 8: Define a constant to represent the seller.
+* Line 9: Define a constant to represent the buyer.
+* Line 10: Finalize participant and other options, and proceed to a Reach step.
+* Line 12: Terminate computation.
+
+# {#wfs-6} Run the DApp
+
+Reach can currently compile the DApp to run on any of the following consensus network types:
+
+* `ALGO`
+* `CFX`
+* `ETH`
+
+Instruct the Reach compiler to connect to a network by setting the `REACH_CONNECTOR_MODE` environment variable.
+One way is to set the variable and execute `reach run` on the same line:
+
+``` nonum
+$ REACH_CONNECTOR_MODE=ALGO reach run
+```
+
+Another is to export the variable and then execute `reach run`:
+
+``` nonum
+$ export REACH_CONNECTOR_MODE=ALGO
+$ ./reach run
+```
+
+Follow these steps to run the DApp:
+
+1. In the VSCode terminal, set `REACH_CONNECTOR_MODE=ALGO` and run the app on the same line:
+
+   ![Run VSCode](/tut/wfs/vscode-run.png)
+
+1. This time, first `export REACH_CONNECTOR_MODE=ALGO`, and then run the app:
+
+    ``` nonum
+    $ export REACH_CONNECTOR_MODE=ALGO-devnet
+    $ ./reach run
+    ```
+
+1. Repeat the first two steps for `CFX` and `ETH`.
+
+> For consistency, output in this module reflects `REACH_CONNECTOR_MODE=ALGO`.
```

Comment:
This is not the first time the word "module" is used to talk about this page, but we don't use that word elsewhere and I think "tutorial" is more used. If you want to change the word, then talk about that with @TheChronicMonster